### ISSUE TITLE
TelnetSpy.cpp:17:21: fatal error: RingBuf.h: No such file or directory

### ISSUE THREAD

**Issue by RE-Specto** [yasheena/telnetspy#12]
```
Compiling .pio\build\esp07\lib6c5\TelnetSpy_ID6236\TelnetSpy.cpp.o
.pio\libdeps\esp07\TelnetSpy_ID6236\TelnetSpy.cpp:17:21: fatal error: RingBuf.h: No such file or directory
......
 #include <RingBuf.h>
                     ^
compilation terminated.
*** [.pio\build\esp07\lib6c5\TelnetSpy_ID6236\TelnetSpy.cpp.o] Error 1

```

looks like a missing undocumented dependency

**Comment by DrJaymz** [comment]
yes, the bog standard example does not compile out of the box.  Which is annoying.

**Comment by yasheena** [comment]
Fixed in version 1.2
